MSProject/ViewControllers/Rentals/WBRentalsViewModel.swift

```diff
@@ -14,18 +14,39 @@ import Networking
 
 class WBRentalsViewModel {
 
+    private var suggestionsBookViewModels: MutableProperty<[WBBookViewModel]> = MutableProperty([])
+
     let repository: WBBooksRepository
     
     init(booksRepository: WBBooksRepository) {
         repository = booksRepository
     }
     
     // MARK: - TableView
-    var numberOfCells: Int {
-        return WBBooksManager.sharedIntance.rentedBooks.value.count
+    var numberOfSections: Int {
+        return 1 + (suggestionsBookViewModels.value.isEmpty ? 0 : 1)
+    }
+    
+    func numberOfCells(for section: Int) -> Int {
+        if section == 1 {
+            return suggestionsBookViewModels.value.isEmpty ? 0 : 1
```

Comment:
If you only need to add one cell you can use the table header instead of adding a whole section just for that cell. it will look much cleaner.